.github/workflows/mac_python_matrix_test.yml

```diff
@@ -0,0 +1,49 @@
+name: MacOS Pyomo Tests Workflow (Slim, Matrixed, Nightly)
```

Comment:
Can we shorten this name and make it more like the other CI providers?  Maybe something like `continuous-integration/github/pr/osx`

(a similar request could be make doe the linux workflow)

Reply:
I changed it for the Mac and the Linux files - so it shows up uniformly.